cypress.json

```diff
@@ -16,6 +16,9 @@
     "password": "admin",
     "ENDPOINT_WITH_PROXY": false,
     "MANAGED_SERVICE_ENDPOINT": false,
-    "VISBUILDER_ENABLED": false
+    "VISBUILDER_ENABLED": false,
+    "DATASOURCE_MANAGEMENT_ENABLED": true,
+    "OSD_TEST_DOMAIN_ENDPOINT_URL": "https://test",
```

Comment:
I'm not sure if it's the best practice to put "OSD_TEST_DOMAIN_ENDPOINT_URL" and "OSD_INVALID_ENPOINT_URL"? are they only used in `1_create_datasources.spec.js`? does it satisfy your use case if you put them into constants.js inside `datasource-management-plugin`?

Reply:
Yes, they are only used for 1 test file. I will move them to `cypress/utils/dashboards/datasource-management-dashboards-plugin/constants.js`